### DISCUSSION TITLE
Transcribed -> Translated -> ... can't export

### DISCUSSION THREAD

**Discussion by jimmyiswhoiam** [ggml-org/whisper.cpp#1909] [General]
Hi, is it possible to export translated text with timestamps?

"Copy Translation" wouldln't include timestamps, and "export" exports with timestamps, but only the original text

I'd be happy for support :)

<img width="910" alt="Zrzut ekranu 2024-02-26 o 16 33 26" src="https://github.com/ggerganov/whisper.cpp/assets/84240387/44a84073-664c-4e9e-b00e-0a1d0c3d38b7">

**Comment by ulatekh** [comment]
What app is this? I didn't even know whisper.cpp had a GUI...
